lib/exercism/use_cases/attempt.rb

```diff
@@ -15,7 +15,7 @@ def initialize(user, *stuff)
   # rubocop:enable Metrics/AbcSize
 
   def valid?
-    !!slug && Xapi.exists?(track, slug)
+    !!slug && Trackler.tracks[track]
```

Comment:
It might be worth implementing a null track in trackler so we can say `Trackler.tracks[track_id].implementations[slug].exists}` e.g.

```
Trackler.tracks["nope"].implementations["nope"].exists?
```

I'll go ahead and look into that.